tests/unit/acl.tcl

```diff
@@ -1007,12 +1036,10 @@ start_server [list overrides [list "dir" $server_path "acl-pubsub-default" "allc
         r ACL setuser harry on nopass resetchannels &test +@all ~*
         r ACL save
 
-        # ACL load will free user and kill clients
         r ACL load
-        catch {r ACL LIST} e
-        assert_match {*I/O error*} $e
```

Comment:
how is this test (`ACL load and save with restricted channels`) different than the one above?
please have a look at the PR that added it and conclude if it should be kept and what to do with it.

Reply:
IIUC this test is checking the behavior of `ACL LOAD`/`SAVE` with users that have restrictions on pub-sub channel access (as opposed to the previous one that just uses default access). See PR #8794 which added this test.